packages/core/src/database/sqlite/service.ts

```diff
@@ -280,6 +280,29 @@ export class SqliteDatabaseService implements BaseDatabaseService {
               )} ago`,
             });
 
+            // Remove any indexes, will be recreated once the app
+            // becomes healthy.
+            await Promise.all(
+              Object.entries(getTables(schema)).flatMap(
+                ([tableName, table]) => {
+                  if (table.constraints === undefined) return [];
+
+                  return Object.keys(table.constraints).map(async (name) => {
+                    await tx
+                      .withSchema(this.userNamespace)
+                      .schema.dropIndex(`${tableName}_${name}`)
```

Comment:
Not sure it matters, but the order of `.schema` and `.withSchema` is different here than in the postgres service.